best-practices/README.md

```diff
@@ -101,6 +101,10 @@ Rails
   environment variables are detected on deploy.
 * [Use blocks][date-block] when declaring date and time attributes in FactoryGirl factories.
 * Use `touch: true` when declaring `belongs_to` relationships.
+* Name service classes after the action they perform
```

Comment:
I don't want to derail this pull request by talking about Method Objects. I've had the same confusion about the `app/services` directory, as there hasn't been much cohesion in "services" within or between projects I've joined, so I think it's good for us to try and firm up this directory. I think your current proposal would definitely introduce some needed structure.

However, I have two concerns about the naming proposed as-is:

* The "services" name seems fairly generic and is used in so many ways that I worry it will be impossible to ever have a group understanding of what "service objects" are, similar to "presenters."
* The current changes seem to very accurately describe a Method Object, and it seems weird to have two names for the same pattern, providing a standard directory for one name.

The way you're describing them reminds me of [Transaction scripts](http://martinfowler.com/eaaCatalog/transactionScript.html). Maybe "transactions" would be a more standard name that would be harder to misinterpret?